quickwit/quickwit-serve/src/ingest_api/rest_handler.rs

```diff
@@ -280,6 +283,25 @@ pub(crate) mod tests {
     use quickwit_proto::ingest::router::IngestRouterServiceClient;
 
     use super::ingest_api_handlers;
+    use crate::ingest_api::lines;
+
+    #[test]
```

Comment:
can you add 2 tests on the handler: one that checks the error message with the input is malformed, and one that checks we get a 200 even with a line with empty space?

Reply:
I added a `200` new test and adjusted the body assertion for `400` test